main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 
 	router := gin.Default()
 
-	docs.SwaggerInfo.BasePath = "/"
+	docs.SwaggerInfo.BasePath = "/api"
```

Comment:
Is the swagger correctly loaded with this change @GhostDragon01 ?
If so looks good to me !

Reply:
Just tested it works, nice job @GhostDragon01 !